templates/server.erb

```diff
@@ -26,11 +26,21 @@ dh <%= @extca_dh_file %>
 crl-verify <%= @extca_ca_crl_file %>
 <%   end -%>
 <% else -%>
+<%- if scope.lookupvar('::openvpn::params::easyrsa_ver') == '2.0' -%>
```

Comment:
please use ```@openvpn::params::easyrsa_ver``` to access variables (or if you are interested you can turn this into an epp template, but that sould probably done in another PR).

Reply:
I need to modify this, so it will be handled.